src/Dfe.PlanTech.Core/Utilities/RecommendationSorter.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Dfe.PlanTech.Core.Contentful.Models;
+using Dfe.PlanTech.Core.DataTransferObjects.Sql;
+using Dfe.PlanTech.Core.Enums;
+using Dfe.PlanTech.Core.Helpers;
+
+namespace Dfe.PlanTech.Core.Utilities;
+
+public static class RecommendationSorter
+{
+    public static List<RecommendationChunkEntry> SortByStatus(
+        this IEnumerable<RecommendationChunkEntry> chunks,
+        Dictionary<string, SqlEstablishmentRecommendationHistoryDto> history,
+        RecommendationSort sortType)
+    {
+        var indexed = chunks.Select((chunk, index) => new { chunk, index });
+
+        return sortType switch
+        {
+            RecommendationSort.Default =>
+                indexed.Select(x => x.chunk).ToList(),
+
+            RecommendationSort.Status =>
+                indexed.OrderBy(x => RecommendationStatusHelper.GetStatus(x.chunk, history))
+                       .ThenBy(x => x.index)
+                       .Select(x => x.chunk)
+                       .ToList(),
+
+            RecommendationSort.LastUpdated =>
+                indexed.OrderByDescending(x => RecommendationStatusHelper.GetLastUpdatedUtc(x.chunk, history))
+                       .ThenBy(x => x.index)
+                       .Select(x => x.chunk)
+                       .ToList(),
```

Comment:
+1 for the fallback on the default sort by index btw!

I did wonder what might happen if several have the same last updated date (e.g. bulk changes after a self assessment) and whether things might jump around, but you've got it covered by also doing the `ThenBy` sort by index!